src/filtering/ugbio_filtering/train_cnv_filtering_model.py

```diff
@@ -0,0 +1,523 @@
+        feature_cols: List of feature columns to use (default: FEATURE_COLS)
+
+    Returns:
+        Tuple of (accuracy_df, curve_df) similar to variant_filtering_utils.eval_model
+    """
+    print(f"\n{data_type} Evaluation:")
+    print("=" * (len(data_type) + 12))
+
+    # Use provided feature columns or default
+    if feature_cols is None:
+        feature_cols = FEATURE_COLS
+
+    # Extract features and labels
+    x = df[feature_cols].copy()
+    y = df["label"].copy()
+
+    # Transform features
+    x_transformed = preprocessor.transform(x)
+
+    # Make predictions
+    y_pred = model.predict(x_transformed)
+    y_pred_proba = model.predict_proba(x_transformed)
+
+    # Convert both labels and predictions to binary for metrics (TP=1, FP=0)
+    y_binary = (y == "TP").astype(int)
+    y_pred_binary = (y_pred == "TP").astype(int)
+
+    # Calculate metrics
+    accuracy = accuracy_score(y_binary, y_pred_binary)
+    precision, recall, f1, _ = precision_recall_fscore_support(y_binary, y_pred_binary, average="binary")
+
+    print(f"Accuracy: {accuracy:.4f}")
+    print(f"Precision: {precision:.4f}")
+    print(f"Recall: {recall:.4f}")
+    print(f"F1-Score: {f1:.4f}")
+
+    print("\nClassification Report:")
+    print(classification_report(y_binary, y_pred_binary))
+
+    print("\nConfusion Matrix:")
+    print(confusion_matrix(y_binary, y_pred_binary))
+
+    # Create accuracy dataframe (similar to variant_filtering_utils format)
+    accuracy_df = pd.DataFrame(
+        {
+            "group": ["ALL"],
+            "accuracy": [accuracy],
+            "precision": [precision],
+            "recall": [recall],
+            "f1": [f1],
+            "support": [len(y_binary)],
+        }
+    )
+
+    # Create curve dataframe with thresholds (simplified version)
+    thresholds = np.linspace(0.1, 0.9, 9)
+    curve_data = []
+
+    for threshold in thresholds:
+        y_pred_thresh = (y_pred_proba[:, 1] >= threshold).astype(int)
+        if len(np.unique(y_pred_thresh)) > 1:  # Avoid division by zero
+            prec = precision_score(y_binary, y_pred_thresh, zero_division=0)
+            rec = recall_score(y_binary, y_pred_thresh, zero_division=0)
+            f1_thresh = f1_score(y_binary, y_pred_thresh, zero_division=0)
+        else:
+            prec = rec = f1_thresh = 0.0
+
+        curve_data.append({"group": "ALL", "precision": prec, "recall": rec, "f1": f1_thresh, "threshold": threshold})
+
+    curve_df = pd.DataFrame(curve_data)
+
+    # Feature importance if available
+    if hasattr(model, "feature_importances_"):
+        print("\nFeature Importance:")
+        # If model has more features than expected, print all
+        feature_names = feature_cols + ["flt"] if len(model.feature_importances_) > len(feature_cols) else feature_cols
+        for name, importance in zip(feature_names, model.feature_importances_, strict=False):
+            print(f"{name}: {importance:.4f}")
+
+    return accuracy_df, curve_df
```

Comment:
suggesting to return also feature importance dataframe since its already calculated here.